index.html

```diff
@@ -6,35 +6,187 @@
 </head>
 <body>
   <header>
-    <img class="logo" src="./elements/lg.png" alt="Logo">
+    <a href="#"><p class="logo">MY LOGO</p></a>
     <nav class="dcp1 navbar">
-        <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
+      <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
     </nav>
   </header>
-  <section class="parent1">   
-      <div class="space"></div>
-      <section class="dcp1 art1">
-          <h1>
-        I’m Sir_Malcom<br>Glad to see<br>you!
-          </h1>
-        </section>
-      <p class="dcp1 art2">
-        I’m a software developer! I can help you build a product, 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hesitate to contact me.
-      </p>
-      <img class="dcp1 social" src="./elements/ltscnnct.jpg" alt="lets connect">
-      <ul class="dcp1 parent2">
-        <li><a href="#" target="_blank"><img src="./elements/gthb.png" alt="github"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/lnkin.png" alt="linkedin"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/twtr.png" alt="twitter"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/m.png" alt="m"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/chill.png" alt="chill"></a></li>
-        </ul>
+  <section class="intro">   
+    <div class="space"></div>
+      <h1 class="dcp1 art1">
+      I’m Sir_Malcom<br>Glad to see<br>you!
+    </h1>
+    <p class="dcp1 art2">
+      I’m a software developer! I can help you build a product, 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hesitate to contact me.
+    </p>
+    <p class="ltscnct">LET’S CONNECT</p>
+    <ul class="dcp1 parent2">
+      <li><a href="#" target="_blank"><img src="./elements/gthb.png" alt="github"></a></li>
+      <li><a href="#" target="_blank"><img src="./elements/lnkin.png" alt="linkedin"></a></li>
+      <li><a href="#" target="_blank"><img src="./elements/twtr.png" alt="twitter"></a></li>
+      <li><a href="#" target="_blank"><img src="./elements/m.png" alt="m"></a></li>
+      <li><a href="#" target="_blank"><img src="./elements/chill.png" alt="chill"></a></li>
+    </ul>
+  </section>
+  <section class="works">
+    <div class="card">
+      <img src="./elements/snpsht.png" alt="Tonic">
+      <div class="prmry-txt">
+            <h2 class="title">Tonic</h2>
+            <div class="frm-2">
+              <span class="clnt">CANOPY</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="role">Back End Dev</span>
+              <img src="./elements/icn.png" alt="circle" class="icon">
+              <span class="year">2015</span>
+            </div>
```

Comment:
[OPTIONAL]

I inspire you to use a `<ul>` tag and `<li>`'s for this section of the projects, although it is not a requirement for this project, it is considered good practice, something that surely will stand out when somebody sees your portfolio 😉

Reply:
@Kakalanp I am unable to implement this suggestion right now due to time constraints. However, I have noted it and will endeavor to do it in the manner suggested